ios/DroidKaigi 2021/Debug.xcodeproj/project.pbxproj

```diff
@@ -14,8 +14,9 @@
 /* End PBXBuildFile section */
 
 /* Begin PBXFileReference section */
-		1ACA166CBF9A421538E309E4 /* DroidKaigiMPP.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = DroidKaigiMPP.framework; path = "../ios-framework/build/xcode-frameworks/DroidKaigiMPP.framework"; sourceTree = "<group>"; };
+		1ACA166CBF9A421538E309E4 /* DroidKaigiMPP.xcframework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.xcframework; name = DroidKaigiMPP.xcframework; path = ../build/xcframeworks/DroidKaigiMPP.xcframework; sourceTree = "<group>"; };
```

Comment:
Currently, this `xcframework` reference is not needed. So please remove this reference 🙏🏼 

Release project too 🙏🏼